pkg/rhcos/builds.go

```diff
@@ -20,12 +20,14 @@ type metadata struct {
 	BaseURI string `json:"baseURI"`
 	Images  struct {
 		QEMU struct {
-			Path   string `json:"path"`
-			SHA256 string `json:"sha256"`
+			Path               string `json:"path"`
+			SHA256             string `json:"sha256"`
+			UncompressedSHA256 string `json:"uncompressed-sha256"`
```

Comment:
Some info in the commit log details on why this change is needed, would be good.